lib/features/contributor/presentation/cubit/contributor_cubit.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:bloc/bloc.dart';
+import 'package:coffeecard/features/contributor/data/datasources/contributor_repository.dart';
+import 'package:coffeecard/features/contributor/domain/entities/contributor.dart';
+import 'package:equatable/equatable.dart';
+
+part 'contributor_state.dart';
+
+class ContributorCubit extends Cubit<ContributorState> {
+  final ContributorRepository repository;
+
+  ContributorCubit({required this.repository})
+      : super(const ContributorLoaded([]));
+
+  Future<void> getContributors() async {
```

Comment:
Method should not be async as there are not any await statements, and no Future is returned